src/applications/vaos/components/layout/CCLayout.jsx

```diff
@@ -72,7 +72,7 @@ export default function CCLayout({ data: appointment }) {
               <Address address={address} />
               <div className="vads-u-margin-top--1 vads-u-color--link-default">
                 <va-icon icon="directions" size="3" srtext="Directions icon" />{' '}
-                <FacilityDirectionsLink location={address} />
+                <FacilityDirectionsLink location={{ address }} />
```

Comment:
Just curious if we need to make the same change on link 72?

Reply:
Good point. I just checked and it fine as is. The 'FacilityDirectionsLink' component expect a struct containing address and 'Address' doesn't.

Reply:
Ah interesting. I need to read up on what the differences between the different types of brackets mean.